From 301f7dcc29c0a648ccb6143b59b5c268db49321d Mon Sep 17 00:00:00 2001
From: Robert Lehmann <robert.lehmann@sitec-systems.de>
Date: Tue, 3 May 2016 15:44:04 +0200
Subject: [PATCH] mach-imx6sx: Add GPIO init for wifi GPIOs

Add initialisation for the following WIFI corresponding GPIOs:
* WIFI_PWR_E
* WIFI_PD
* WIFI_ANT_SEL

Ticket #1136
---
 arch/arm/mach-imx/mach-imx6sx.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/mach-imx/mach-imx6sx.c b/arch/arm/mach-imx/mach-imx6sx.c
index 256dee6..c5e85fc 100755
--- a/arch/arm/mach-imx/mach-imx6sx.c
+++ b/arch/arm/mach-imx/mach-imx6sx.c
@@ -336,6 +336,9 @@ static void gpio_request_export(int gpio_num, int def_mode, char *label)
 #define GSM_ANT_SEL IMX_GPIO_NR(3, 25)
 #define UART6_CTS IMX_GPIO_NR(2, 15)
 #define UART6_RTS IMX_GPIO_NR(2, 10)
+#define WIFI_PWR_E IMX_GPIO_NR(3, 26)
+#define WIFI_PD IMX_GPIO_NR(1, 11)
+#define WIFI_ANT_SEL IMX_GPIO_NR(3, 16)
 
 static void imx6sx_gpio_setup_s2(void)
 {
@@ -350,6 +353,9 @@ static void imx6sx_gpio_setup_s2(void)
     gpio_request_export(GSM_ANT_SEL, GPIOF_OUT_INIT_HIGH, "gsm_ant_sel");
 	gpio_request_export(UART6_RTS, GPIOF_IN, "uart6_cts");
 	gpio_request_export(UART6_CTS, GPIOF_OUT_INIT_LOW, "uart6_rts");
+	gpio_request_export(WIFI_PWR_E, GPIOF_OUT_INIT_HIGH, "wifi_pwr_e");
+	gpio_request_export(WIFI_PD, GPIOF_OUT_INIT_LOW, "wifi_pd");
+	gpio_request_export(WIFI_ANT_SEL, GPIOF_OUT_INIT_LOW, "wifi_ant_sel");
     return;
 }
 
-- 
1.9.1

